mongodb-atlas-config.yml.sample

```diff
@@ -0,0 +1,11 @@
+integrations:
```

Comment:
Could we add a comment here explaining the caveats of monitoring Atlas?

Something like:

```suggestion
# This file shows an example config targeting a MongoDB Atlas shard. Please note that:
# - An instance of the integration is required to monitor each shard individually
# - Direct shard access is only available on tier ??? or above, so tiers ??? and below are not supported for this configuration.
integrations:
```

Reply:
Patched